ledger/ledger-api-auth/src/test/suite/scala/com/digitalasset/ledger/api/auth/AuthorizationInterceptorSpec.scala

```diff
@@ -57,12 +61,15 @@ class AuthorizationInterceptorSpec
       AuthorizationInterceptor(
         authService,
         Some(userManagementService),
+        identityProviderAwareAuthService,
         global,
       )
 
     val statusCaptor = ArgCaptor[Status]
     val metadataCaptor = ArgCaptor[Metadata]
 
+    when(identityProviderAwareAuthService.checkForIdpAwareAccess(any[Metadata]))
```

Comment:
Would be great not to need mocks and then drop the `IdentityProviderAwareAuthServiceImpl` and `IdentityProviderAwareAuthService` distinction but it's fine as also